src/app/query/bitcoin/ordinals/use-taproot-address-utxos.query.ts

```diff
@@ -15,6 +15,11 @@ export interface TaprootUtxo extends UtxoResponseItem {
   addressIndex: number;
 }
 
+export interface NativeSegwitUtxo extends UtxoResponseItem {
+  // not sure if this is right, but noticed that this is always undefined for ns
+  addressIndex: undefined;
+}
```

Comment:
Elsewhere in the code, I added this to `TrUtxo` as we needed to know which address index it's on, in `useTaprootAccountUtxosQuery`.

Can/should we do the same for native segwit?

Reply:
yes, we can. added index for ns as well in the latest commit